src/sentry/static/sentry/app/components/progressBar.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Tooltip from 'app/components/tooltip';
+import {Color} from 'app/utils/theme';
+
+type Props = {
+  /**
+   * The value of the progress indicator for the determinate variant. Value between 0 and 100
+   */
+  value: number;
+  /**
+   * The color of the component.
+   */
+  progressColor?: Color;
```

Comment:
Should you also take `Alias` from the theme module? Having named aliases will work better with darkmode.